### DISCUSSION TITLE
TypeError dispatch(livewire/livewire)

### DISCUSSION THREAD

**Discussion by ricardo-lobo** [livewire/livewire#4906] [7. Bugs (archived)]
Hello.

I am getting lots of errors reported to Sentry will the following info:

> TypeError: Illegal invocation
>   at HTMLDocument.document.createEvent(:1:40420)
>   at dispatch(/livewire/livewire.js:1:9053)
>   at ? (/livewire/livewire.js:13:148618)
>   at ? (/livewire/livewire.js:1:232)
>   at ? (/livewire/livewire.js:1:242)

The only thing in common between all of these reports (hundreds) is the fact that it only seems to be happening in mobile browsers eg. Mobile Safari, Chrome Mobile

Are there any polyfills required to solve this issue? Or this not an issue at all?

**Comment by levu42** [comment]
I'm seeing the same thing for mobile browsers

**Comment by NBA707** [comment]
Seems like a known issue with iOS devices in how the dispatch event function works based on googling.

**Comment by joshhanley** [comment]
Hmm that isn't good! Can someone provide a sample component class and view that triggers this error so we can investigate?

Thanks!

**Reply by ricardo-lobo** [reply]
@joshhanley cannot provide a sample. We can maybe schedule a call and I'll share our Sentry and/or App Source with you

update:

While checking Sentry, we haven't had any errors of this kind for the last 30 days. Don't know if some dependency has been updated that solved this issue.

**Reply by joshhanley** [reply]
@ricardo-lobo sorry, I'm not available for calls at the moment as I do this in my spare time.

If you can work out what triggers the issue in your repo and setup a new repo that only demonstrates the issue, then I'd be happy to have a look 🙂

Ah just read your update after typing this out haha. All good!

But yeah this also applies to anyone else, if someone can share an example repo demonstrating the issue, I'd be happy to have a look 🙂
